src/visit_vtk/full/vtkVisItPolyDataNormals2D.C

```diff
@@ -286,7 +314,7 @@ vtkVisItPolyDataNormals2D::ExecuteCell(vtkPolyData *input, vtkPolyData *output)
     // The verts come before the polys.  So add normals for them.
     int numPrimitivesWithoutNormals = 0;
     numPrimitivesWithoutNormals += input->GetVerts()->GetNumberOfCells();
```

Comment:
Does it matter we're adding `int` and `vtkIdType` here? It almost feels like `numPrimitivesWithoutNormals` should be `vtkIdType` instead of `int`.

Reply:
I made use of vtkIdType consistent in this file.